agpypeline/entrypoint.py

```diff
@@ -419,5 +419,14 @@ def entrypoint(configuration_info: Configuration, algorithm_instance: Algorithm)
         configuration_info: an instance of Configuration class
         algorithm_instance: an instance of class for preparing work
     """
+    begin_process(configuration_info, algorithm_instance)
```

Comment:
Why was this broken apart into a separate function? Is this in preparation for renaming?

Reply:
It was broken apart because of [this ZenHub issue](https://app.zenhub.com/workspaces/ua-ag-data-science-5a57a3198339f11ba1c85775/issues/agpipeline/issues-and-projects/188)